packages/ckeditor5-engine/src/model/model.ts

```diff
@@ -3,16 +3,18 @@
  * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
  */
 
+/* eslint-disable @typescript-eslint/unified-signatures */
```

Comment:
Why this is required? Maybe it could be located next to the affected line if it's really needed?

Reply:
They are here to make `createSelection` method more readable.